pom.xml

```diff
@@ -32,7 +28,7 @@
         <dependency>
             <groupId>org.openapitools</groupId>
             <artifactId>openapi-generator-core</artifactId>
-            <version>${openapi-generator-maven-plugin-version}</version>
+            <version>7.1.0</version>
```

Comment:
Remove this dependency! (as stated in previous comments)

Reply:
I'm so sorry for the misunderstanding! 
I misunderstood what needed to be changed, but now I believe it is fixed for real this time.